angular.json

```diff
@@ -27,13 +27,24 @@
                 "glob": "**/*",
                 "input": "src/public",
                 "output": "/"
+              },
+              {
+                "glob": "**/*",
+                "input": "node_modules/@hmcts/media-viewer/assets",
+                "output": "/assets"
               }
             ],
             "styles": [
               "src/style/app.scss",
-              "node_modules/govuk_template_mustache/assets/stylesheets/govuk-template.css"
+              "node_modules/govuk_template_mustache/assets/stylesheets/govuk-template.css",
```

Comment:
i wouldn't expect these to be needed directly here to use Media Viewer.

Reply:
Done.